channel/MonsterChannel.m

```diff
@@ -308,44 +307,11 @@ function setupQuadrigaLayout(obj, Stations, Users)
 						LOS = obj.fresnelLOScomputation(Station, User, draw);
 						prop = NaN;
 					case '3GPP38901-probability'
-						[LOS, prop] = Monster3GPP38901.LOSprobability(obj, Station, User);
-					case 'NLOS'
-						LOS = 0;
-						prop = NaN;
-					case 'LOS'
-						LOS = 1;
-						prop = NaN;
-				end
-			end
-		end
-		
-		
-		function [LOS, prop] = isLinkLOSMatrix(obj, Station, User, draw, dist2d)
-			% Check if link between `txPos` and `rxPos` is LOS using one of two methods
-			%
-			% 1. :attr:`SonohiChannel.LOSMethod` : :attr:`fresnel` 1st Fresnel zone and the building footprint.
-			% 2. :attr:`SonohiChannel.LOSMethod` : :attr:`3GPP38901-probability` Uses probability given table 7.4.2-1 of 3GPP TR 38.901. See :meth:`ch.SONOHImodels.3GPP38901.sonohi3GPP38901.LOSprobability` for more the implementation.
-			%
-			% :param Station: Need :attr:`Stations.Position` and :attr:`Stations.DlFreq`.
-			% :type Station: :class:`enb.EvolvedNodeB`
-			% :param User: Need :attr:`User.Position`
-			% :type User: :class:`ue.UserEquipment`
-			% :param bool draw: Draws fresnel zone and elevation profile.
-			% :returns: LOS (bool) indicating LOS
-			% :returns: (optional) probability is returned if :attr:`3GPP38901-probability` is assigned
-			
-			% Check if User is indoor
-			% Else use probability to determine LOS state
-			if User.Mobility.Indoor
-				LOS = 0;
-				prop = NaN;
-			else
-				switch obj.LOSMethod
-					case 'fresnel'
-						LOS = obj.fresnelLOScomputation(Station, User, draw);
-						prop = NaN;
-					case '3GPP38901-probability'
-						[LOS, prop] = Monster3GPP38901.LOSprobabilityMatrix(obj, Station, User, dist2d);
+						if isempty(varargin)
```

Comment:
I think we need to reconsider this. The passing of varargin might be too nested. And it needs some description.